plane/source/docs/guide-tilt-rotor.rst

```diff
@@ -221,6 +221,26 @@ degrees that the tilt mechanism can go.
 You also need to setup your two tilt servos with SERVOn_FUNCTION=75
 for left tilt and SERVOn_FUNCTION=76 for right tilt.
 
+Non-Vectored Yaw
+================
+
+Non-Vectored yaw aircraft (Q_TILT_TYPE 0 or 1) needs a tilt servo for the rear
+motor for yaw control.
+
+You need to setup your two front tilt servos both with SERVOn_FUNCTION=41
+and also your rear servo for yaw control with SERVOn_FUNCTION=39. 
+FUNCTION=39 is normally the servofunction for motor 7. Ony in non-vectored
+tri-copter the yaw servo is controlled via SERVOn_FUNCTION 39.
+
+Note that if you want to use BLHELI ESCs with telemetry in non-vectored tri-copter,
+you must not set BLHELI to 1. Set instead SERVO_BLH_MASK to the output-bitmask
```

Comment:
SERVO_BLH_AUTO is correct. 

Sorry, I didn't know how to write down the hotlinks I promise improvement

"and are you certain that SERVO_BLH_AUTO will select 39 as a BLHELI ESC even with Tricopter frame class selected?" see above

Reply:
unfortunately I cant really show you the correct way here since this response box interprets the back-ticks as a different markup element....see the Wiki Style guide for the details on the hotlink formats...